src/huggingface_hub/hf_api.py

```diff
@@ -909,8 +920,10 @@ def list_metrics(self) -> List[MetricInfo]:
         d = r.json()
         return [MetricInfo(**x) for x in d]
 
+    @_deprecate_positional_args
     def model_info(
         self,
+        *,
         repo_id: str,
```

Comment:
```suggestion
    @_deprecate_positional_args
    def model_info(
        self,
        repo_id: str,
        *,
```

Reply:
So here the issue is consistency. Having `repo_id` as positional (which I agree makes sense) is inconsistent with {create,delete}_repo` and `update_repo_visibility`.

Reply:
actually with your proposal in https://github.com/huggingface/huggingface_hub/pull/733#pullrequestreview-903141631 this should work.